frontend/src/scenes/teamActivityDescriber.tsx

```diff
@@ -37,6 +37,17 @@ const teamActionsMapping: Record<
             ],
         }
     },
+    session_recording_url_trigger_config(change: ActivityChange | undefined): ChangeMapping | null {
+        const before = change?.before
+        const after = change?.after
+        if (before === null && after === null) {
+            return null
+        }
+
+        return {
+            description: [<>Changed session replay URL triggers</>],
```

Comment:
not blocking since I'm a fan of small PRs

but we can compare the before and after arrays to give some more useful info here

Reply:
Agreed, I'll add that in the follow up: the lack of types made it a bit harder to figure out what's going on, so I just added a placeholder to make it compile